spec/requests/api/arbitration_profile_spec.rb

```diff
@@ -42,6 +42,81 @@
       expect(response).to have_http_status(:ok)
     end
 
+    it 'supports arbitration_default creation with provider href or id specified' do
+      api_basic_authorize collection_action_identifier(:arbitration_profiles, :create)
+      provider = FactoryGirl.create(:ext_management_system)
+      provider_href = providers_url(provider.id)
+      request_json = {:name => 'arbitration profile'}
+
+      expect do
+        run_post(arbitration_profiles_url, request_json.merge(:provider => {:href => provider_href}))
+      end.to change(ArbitrationProfile, :count).by(1)
+
+      expect do
+        run_post(arbitration_profiles_url, request_body.merge(:provider => {:id => provider.id}))
```

Comment:
While I know these tests are expensive, I might lean towards having a separate test per request, as you're testing a different behavior here. But I'd be curious to see what others think. It would be awesome if we could check that href/id parsing works in a separate unit test, but that might not be possible yet! It's unfortunate that you have to go through the whole request cycle to check that both id/hrefs work, when everything else is exactly the same.